cost-analyzer/templates/aggregator-cloud-cost-deployment.yaml

```diff
@@ -149,10 +149,10 @@ spec:
       {{- end }}
       containers:
         {{- include "aggregator.cloudCost.containerTemplate" . | nindent 8 }}
-    {{- if .Values.imagePullSecrets }}
```

Comment:
These `if` statements should remain or use a `with` statement so that `imagePullSecrets:` is only provided if any are listed in values. If none are used (the default) the key shouldn't be templated in.

Reply:
Etc. elsewhere

Reply:
Add if statements back as suggested.

Reply:
cc @chipzoller